src/Leaflet.Renderer.Canvas.Tile.js

```diff
@@ -52,7 +52,7 @@ L.Canvas.Tile = L.Canvas.extend({
 			}
 		}
 		if (clickedLayer)  {
-			L.DomEvent.fakeStop(e);
+			L.DomEvent._fakeStop(e);
```

Comment:
No, we want to keep `fakeStop()` for compatibility with the `master` branch of Leaflet. If you are going to have it back, it should be as a fallback.